Jenkinsfile

```diff
@@ -20,6 +20,117 @@
 /* groovylint-disable-next-line CompileStatic */
 job_status_internal = [:]
 
+/**
```

Comment:
I needed to edit the unitTestPost function so I just moved the whole thing as it belongs in the DAOS repo anyway and doesn't affect the bytecode limit.